lib/handin/accounts.ex

```diff
@@ -360,4 +360,40 @@ defmodule Handin.Accounts do
     |> Map.get(:modules)
     |> Enum.any?(&(&1.id == module_id))
   end
+
+  def list_users() do
+    Repo.all(User)
+  end
+
+  def list_users(params) do
+    case Flop.validate_and_run(User, params, for: User) do
+      {:ok, {users, meta}} ->
+        %{users: users |> Enum.map(&Repo.preload(&1, :university)), meta: meta}
+
+      {:error, meta} ->
+        %{users: [], meta: meta}
+    end
+  end
+
+  def edit_changeset(user, attrs \\ %{}) do
+    user
+    |> Repo.preload(:university)
+    |> User.edit_changeset(attrs)
+  end
+
+  def update_user(user, attrs) do
+    case user
+         |> edit_changeset(attrs)
+         |> Repo.update() do
+      {:ok, user} ->
+        {:ok, Repo.preload(user, :university, force: true)}
```

Comment:
eta `force: true` garna parcha ra?

Reply:
force true nagarda xa ni university_id of another university is mapped but the association remains the old one for some reason